### ISSUE TITLE
tvOS Support

### ISSUE THREAD

**Issue by plashchynski** [magicalpanda/MagicalRecord#1218]
Any chance?

**Comment by hassaan4584** [comment]
Although its a little late, still...
I have recently successfully added MagicalRecord as a submodule in a tvOS project.

**Comment by bhatti-waqas** [comment]
@hassaan4584 ! Its good you added it in project as submodule. But were you able to run it on Reas tv-OS device ?

**Comment by hassaan4584** [comment]
As a matter of fact I wasn't able to. 
While enabling exception breakpoint, when i tried to run it on the real device it would throw me an exception saying that I don't have rights or something to create a file.

**Comment by bhatti-waqas** [comment]
Hmm. So I guess we need to use Core-Data by ourself instead of Magical Record. Thats sad. :(

**Comment by jordanbigel** [comment]
I've been using MR in a shipping tvOS app for about a year. Works fine. Can't remember exactly what I had to do to it to get it to work, but it does work. I Know the last comment on this thread was a year ago, but I came back here to see if there was any "official" tvOS version yet. Apparently not. So, if anyone wants to use my version, I've posted it to a public GitHub. I don't usually do this, but I cant believe how long its been since tvOS has been available and still no update from magicalPanda. Kinda worries me since I've got 2 apps both relying on MR extensively. What is it going to take for Panda to support tvOS? Or is MR already a dead project? https://github.com/jordanbigel/MagicalRecord-tvOS
